README.md

```diff
@@ -1 +1,71 @@
-# eip-draft
\ No newline at end of file
+# EIP-XXXX: Cryptographic Proof of Exchange Solvency
+
+## Preamble
+
+```
+EIP: XXXX
+Title: Cryptographic Proof of Exchange Solvency
+Author: [ADD NAMES HERE]
+Type: Standards Track (ERC)
+Category: ERC
+Status: Draft
+Created: 2023-08-30
+```
+
+## Abstract
+
+This EIP proposes a standard for centralized exchanges to cryptographically prove their solvency without revealing confidential information about user balances using cryptographic techniques like zero-knowledge proofs.
+
+## Motivation
+
+The lack of a standardized, cryptographic verification process for exchange solvency puts users at risk and lacks transparency. This standard aims to provide a reliable, auditable, and decentralized mechanism for verifying exchange solvency.
+
+## Specification
+
+### Solidity Interface
+
+```solidity
+// [Sample Interface showing methods for proof of solvency]
+```
+
+### Core Circuits
+
+- **Proof of Inclusion Circuit**: Ensures a user's assets are included in the total calculation of the exchange's assets.
+- **Proof of Solvency Circuit**: Verifies the solvency of the exchange.
+
+### Data Structures
+
+- **Liability Tree**: Data structure used to maintain and compute liabilities.
+- **Asset Sum Tree**: Data structure used for aggregating assets.
+
+### Off-Chain Components
+
+[Add off-chain components here]
```

Comment:
I think that for now, Solidity Interface + Circuits + Data Structures are enough. We have some backend code that contains the exectuables to generate and verify the proof but we can just mentione in the reference implementation section